articles/confidential-ledger/manage-certificate-based-users.md

```diff
@@ -0,0 +1,243 @@
+---
```

Comment:
Since this doc is fairly similar to the AAD Token doc, I have the same comments related to missing links and suggested changes on this doc. Feel free to review any of them, thanks.